tests/Integration/ModulesIntegrationTest.php

```diff
@@ -20,8 +19,6 @@ public function testInvoke(): void
 
         $modules = $event->getModules();
 
-        $this->assertArrayHasKey('sentry/sentry', $modules, 'Root project missing');
-        $this->assertArrayHasKey('ocramius/package-versions', $modules, 'Indirect dependency missing');
-        $this->assertEquals(PrettyVersions::getVersion('sentry/sentry'), $modules['sentry/sentry']);
+        $this->assertNotEmpty($modules);
```

Comment:
Why did you suppress everything here? Last line could survive.

Reply:
Yes, it could. However I don't think that it's worth keeping it because it doesn't add any value to the test and also because I don't want to the test to depend anymore on the names of the dependencies or of the root package because if those changes then our build shoud not break like it did in the past